orchard-core/src/main/scala/com/salesforce/mce/orchard/OrchardSettings.scala

```diff
@@ -36,6 +39,8 @@ class OrchardSettings private (config: Config) {
 
   val resourceReattemptDelayPolicy = delayPolicy(config, "resource.reAttemptDelay")
 
+  val workflowTtl = Try(config.getDuration("workflow.ttl")).getOrElse(Duration.ofDays(-1))
```

Comment:
why is `Try` required? You already have it configured in reference, no?

Reply:
it is compatible, if you get this new version, you get a new `reference.conf`, so the `Try` will never fail unless it's miss configured with the the wrong data type with parsing error